test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/PatchTests.cs

```diff
@@ -95,5 +95,141 @@ public async Task GivenAServerThatSupportsIt_WhenSubmittingAForbiddenPropertyPat
 
             Assert.Equal(HttpStatusCode.BadRequest, exception.Response.StatusCode);
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenAServerThatSupportsIt_WhenSubmittingAReplacePatch_ThenServerShouldPatchCorrectly()
+        {
+            var poco = Samples.GetDefaultPatient().ToPoco<Patient>();
+            FhirResponse<Patient> response = await _client.CreateAsync(poco);
+            Assert.Equal(AdministrativeGender.Male, response.Resource.Gender);
+            Assert.NotNull(response.Resource.Address);
+
+            string patchDocument =
+                "[{\"op\":\"replace\",\"path\":\"/gender\",\"value\":\"female\"}]";
+
+            using FhirResponse<Patient> patch = await _client.PatchAsync(response.Resource, patchDocument);
+
+            Assert.Equal(HttpStatusCode.OK, patch.Response.StatusCode);
+            Assert.Equal(AdministrativeGender.Female, patch.Resource.Gender);
+        }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenAServerThatSupportsIt_WhenSubmittingAddBirthdatePatch_ThenServerShouldPatchCorrectly()
+        {
+            var poco = Samples.GetDefaultPatient().ToPoco<Patient>();
+            FhirResponse<Patient> response = await _client.CreateAsync(poco);
+            string patchDocument =
+                "[{\"op\":\"add\",\"path\":\"/birthDate\",\"value\":\"1970-12-25\"}]";
+
+            using FhirResponse<Patient> patch = await _client.PatchAsync(response.Resource, patchDocument);
+
+            Assert.Equal(HttpStatusCode.OK, patch.Response.StatusCode);
+            Assert.Equal("1970-12-25", patch.Resource.BirthDate);
+        }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task
+            GivenAServerThatSupportsIt_WhenSubmittingReplaceBirthdatePatch_ThenServerShouldPatchCorrectly()
```

Comment:
Rename test to something generic instead of using the property name "ReplaceBirthdatePatch" -> WhenReplacingPropertyPatch_Then..."

Reply:
Consolidated these in single test!